src/include/btmem.h

```diff
@@ -871,6 +893,7 @@ WT_PACKED_STRUCT_BEGIN(__wt_update)
 	 */
 #define	WT_UPDATE_DELETED_ISSET(upd)	((upd)->size == UINT32_MAX)
 #define	WT_UPDATE_DELETED_SET(upd)	((upd)->size = UINT32_MAX)
+#define	WT_UPDATE_DELETED_VALUE		UINT32_MAX
```

Comment:
Maybe change `WT_UPDATE_DELETED_ISSET` and `WT_UPDATE_DELETED_SET` in terrms of `WT_UPDATE_DELETED_VALUE`.